.github/pull_request_template.md

```diff
@@ -8,7 +8,11 @@
 If any of these checks are missing, please provide an explanation.
 
 - [ ] I have updated README files, if applicable.
-- [ ] I have run `make pre-pr` and all checks have passed.
+- [ ] I have run `make pre-pr`, or the on-pr workflow tests and on-schedule
+  tests the and all checks have passed.
```

Comment:
I also updated this item in the PR checklist as `make pre-pr` isn't really helpful locally if you don't have all the API tokens, and it would be sufficient to have the `make test` and `make test-live` checks run in CI, the former by the "on-prs.yml" workflow and the latter triggered manually via the "on-schedule.yml" workflow